inferno-core/src/Inferno/Infer.hs

```diff
@@ -381,7 +381,7 @@ inferTypeReps allTypeClasses (ForallTC tvs tyCls (ImplType _impl ty)) inputTys o
         Left errs -> Left errs
         Right subst ->
           let tyClsSubst = Set.map (apply subst) tyCls
-           in case find (\(TypeClass nm _) -> nm == "rep") $ Set.toList tyClsSubst of
+           in case List.find (\(TypeClass nm _) -> nm == "rep") $ Set.toList tyClsSubst of
```

Comment:
hmm not sure why i wasn't just using set find here?? probably not terribly important tho